### PR TITLE
initial log-spammer work

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:

- add log spammer 

## Security considerations

None

### CODE REVIEW

File: spam.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import logging
+import os
+import time
+
+
+logging.basicConfig(format="%(asctime)s %(message)s")
+
+
+def main():
+    my_guid = os.getenv("CF_INSTANCE_GUID", "¯\_(ツ)_/¯")
```
Comment (ccostino) [inline spam.py:11]: 😆 Seriously question though, will the use of the emoji cause any issues?

**Review by ccostino — COMMENTED** [review]
Quick question to make sure things work fine. :-)

File: spam.py (hunk 1 above)
Comment (bengerman13) [reply spam.py:11]:
```
[log-spammer](intial-features)$ python spam.py
 7 ---
2020-05-22 15:15:47,788 ¯\_(ツ)_/¯ - 0
2020-05-22 15:15:48,291 ¯\_(ツ)_/¯ - 1
2020-05-22 15:15:48,795 ¯\_(ツ)_/¯ - 2
```
works on my machine  ¯\\\_(ツ)_/¯

# Serious answer
[python3 uses utf-8 as the default encoding for source code](https://www.python.org/dev/peps/pep-3120/)

Comment (ccostino) [reply spam.py:11]: Fair enough! :grinning:

**Review by ccostino — APPROVED** [review]
Thanks, @bengerman13!

**Pull request merged**